ghost/admin/app/components/members/modals/logout-member.js

```diff
@@ -0,0 +1,31 @@
+import Component from '@glimmer/component';
+import {inject as service} from '@ember/service';
+import {task} from 'ember-concurrency';
+
+export default class LogoutMemberModal extends Component {
+    @service notifications;
+    @service ajax;
+    @service ghostPaths;
+
+    get member() {
+        return this.args.data.member;
+    }
+
+    @task({drop: true})
+    *logoutMemberTask() {
+        try {
+            const url = this.ghostPaths.url.api('/members/', this.member.id, '/logout/');
+            const options = {};
+            yield this.ajax.post(url, options);
```

Comment:
Should we be using a `DELETE` request instead of `POST` for logging out, i.e. deleting a session?

Reply:
I'll change it to `DELETE /members/id/sessions/`

Reply:
✅